scripts/compose.py

```diff
@@ -1051,6 +1053,19 @@ def __init__(self, **options):
                 self.environment["ELASTICSEARCH_PASSWORD"] = "changeme"
                 self.environment["ELASTICSEARCH_USERNAME"] = "kibana_system_user"
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
+        self.es_urls = self.options.get("kibana_elasticsearch_urls")
```

Comment:
maybe more pythonic to write it as and remove the 2 following lines
```suggestion
        self.es_urls = self.options.get("kibana_elasticsearch_urls") or ["http://elasticsearch:9200"]
```